profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/tmgmt_poetry.install

```diff
@@ -85,18 +85,96 @@ function tmgmt_poetry_schema() {
 }
 
 /**
+ * Implements hook_requirements().
+ */
+function tmgmt_poetry_requirements($phase) {
+  $requirements = array();
+  $t = get_t();
+  global $conf;
+
+  switch ($phase) {
+    case 'install':
+      if (!isset($conf['poetry_service']['address'])) {
+        $requirements['poetry_service'] = array(
+          'title' => 'DGT connector webservice',
+          'severity' => REQUIREMENT_ERROR,
+          'description' => $t('The DGT webservice endpoint is not set.  Please ask your support team to check the server configuration.'),
+          'value' => $t('Not configured'),
+        );
+      }
+      else {
+        $requirements['poetry_service'] = array(
+          'title' => $t('DGT connector webservice'),
+          'severity' => REQUIREMENT_OK,
+          'value' => $t('The DGT endpoint is correctly set'),
+        );
+      }
+      break;
+
+    case 'runtime':
```

Comment:
But indeed this part needs to stay in the tmgmt_poetry.install as it is related to the data created during installation of the tmgmt_poetry module.